src/controllers/todoController.ts

```diff
@@ -0,0 +1,29 @@
+import Todo from "models/todoModel";
+import { Request, Response } from "express";
+import expressAsyncHandler from "express-async-handler";
+
+export const getTodos = expressAsyncHandler(async (req: any, res: Response) => {
+  const todos: any = await Todo.find({ user: req.user._id });
+  res.status(200).json(todos);
```

Comment:
This should return 404 if todo is not found. Also, handle all such cases in the controllers.